Cargo.toml

```
@@ -58,18 +57,26 @@ parser = [
     "bytes",
     "chrono",
     "env_logger",
-    "hex",
     "log",
     "models",
     "oneio",
     "regex",
-    "serde_json",
 ]
 cli = [
     "clap",
     "parser",
     "env_logger"
```

```suggestion
    "env_logger",
    "serde",
    "serde_json"
```
Add `serde` and `serde_json` for building cli.